integration-tests/src/test/java/io/testerra/test/pretest_status/GenerateSimpleTestStatusTest.java

```diff
@@ -0,0 +1,51 @@
+package io.testerra.test.pretest_status;
+
+import eu.tsystems.mms.tic.testframework.execution.testng.AssertCollector;
+import eu.tsystems.mms.tic.testframework.execution.testng.OptionalAssert;
+import eu.tsystems.mms.tic.testframework.report.Status;
+import eu.tsystems.mms.tic.testframework.testing.TesterraTest;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+public class GenerateSimpleTestStatusTest extends TesterraTest {
+
+    private final String SKIPPED_EXCEPTION_MESSAGE = String.format("Test %s.", Status.SKIPPED.title);
+
+    @Test
+    public void test_Passed() {
+        Assert.assertTrue(true, "Creating TestStatus 'Passed'");
+    }
+
+    @Test
+    public void test_Failed() {
+        Assert.fail("Creating TestStatus 'Failed'");
+    }
+
+    @Test
+    public void test_Optional_Assert() {
+        OptionalAssert.fail("minor fail");
+    }
+
+    @Test
+    public void test_SkippedNoStatus() {
+        throw new SkipException(SKIPPED_EXCEPTION_MESSAGE);
```

Comment:
I kind of expect this one to differ from `test_Skipped` below? Names read as if the latter is missing something?

Reply:
Dublicate method...